src/server/acl/acl_family.cc

```diff
@@ -124,19 +144,22 @@ std::pair<OptCat, bool> MaybeParseAclCategory(std::string_view command) {
 
 using facade::ErrorReply;
 
-std::variant<User::UpdateRequest, ErrorReply> ParseAclSetUser(CmdArgList args) {
+template <typename T>
+std::variant<User::UpdateRequest, ErrorReply> ParseAclSetUser(T args, bool hashed = false) {
```

Comment:
Small nit: Maybe use `absl::Span<T>`

Just a random thought, using the parser would solve the problem because it could hide the mutability (if an argslice constructor is added)

Reply:
I will refactor it at some point, I am expecting to add some changes here so I don't want to refactor right now